packages/rulesets/src/spectral/functions/delete-response-codes.ts

```diff
@@ -0,0 +1,29 @@
+export const DeleteResponseCodes = (deleteOp: any, _opts: any, ctx: any) => {
+  if (deleteOp === null || typeof deleteOp !== "object") {
+    return []
+  }
+  const path = ctx.path
+  const errors = []
+
+  if (!deleteOp?.responses) {
+    return []
+  }
+
+  if (deleteOp["x-ms-long-running-operation"] && deleteOp["x-ms-long-running-operation"] === true) {
+    if (!deleteOp.responses["202"] || !deleteOp.responses["204"]) {
```

Comment:
>deleteOp.responses["204"] [](http://example.com/codeflow?start=39&length=25)

lets also check that no other status code is specified

Reply:
@rkmanda should I have the rule ensure `202`, `204`, and `default` are all specified and no others for LROs?